### PR TITLE
Do not stop WF before all tickets are closed

### PR DESCRIPTION
**Pull request by pazembrz**

### CODE REVIEW

File: inspirehep/modules/workflows/workflows/article.py [hunk 1]
```diff
@@ -349,12 +349,10 @@
                         [
                             reject_record('Article automatically rejected'),
                             mark('approved', False),
-                            save_workflow,
-                            stop_processing,
                         ],
-                    )
-                )
-            ]
+                    ),
```
Comment (drjova) [inline inspirehep/modules/workflows/workflows/article.py:353]: we don't have to `save_workflow`?

Comment (pazembrz) [reply inspirehep/modules/workflows/workflows/article.py:353]: we can but we don't have to as wf is not stopped. It's saved literally 3 lines below and this executes for all paths.

File: inspirehep/modules/workflows/workflows/article.py [hunk 2]
```diff
@@ -349,12 +349,10 @@
                         [
                             reject_record('Article automatically rejected'),
```
Comment (michamos) [inline inspirehep/modules/workflows/workflows/article.py:350]: this is incorrect, you reach this also if it's manually rejected and not for a hidden collection.

Comment (pazembrz) [reply inspirehep/modules/workflows/workflows/article.py:350]: Do we need an exact reason here or can I just put "Article rejected"?

Comment (michamos) [reply inspirehep/modules/workflows/workflows/article.py:350]: no, that's fine

**Review by michamos — APPROVED** [review]

**Pull request merged**